charts/selenium-grid/values.yaml

```diff
@@ -621,6 +637,12 @@ edgeNode:
   # - name: my-extra-volume-from-pvc
   #   persistentVolumeClaim:
   #     claimName: my-pv-claim
+  # Keda scaled object configuration
+  autoscalingEnabled: false
+  maxReplicaCount: 8
+  hpa:
+    url: http://selenium-hub.default:4444/graphql # Repalce your http graphql url here
+    browserName: MicrosoftEdge
```

Comment:
If you was Edge to work you will need to have "sessionBrowserName" as well set to "msedge"

Docs: https://keda.sh/docs/2.8/scalers/selenium-grid-scaler/
Issue regarding this: https://github.com/kedacore/keda/issues/2709